src/views/admin/PersonDetailsView.vue

```diff
@@ -326,9 +343,14 @@
 
   // This will send the updated list of Zuordnungen to the Backend on TOP of the new added one through the form.
   const confirmAddition = async (): Promise<void> => {
-    await personenkontextStore.updatePersonenkontexte(finalZuordnungen.value, currentPersonId);
-    createSuccessDialogVisible.value = true;
-    resetForm();
+    try {
+      await personenkontextStore.updatePersonenkontexte(finalZuordnungen.value, currentPersonId);
+      createSuccessDialogVisible.value = true;
+      resetForm();
+    } catch {
+      creationErrorText.value = t(`admin.personenkontext.errors.${personenkontextStore.errorCode}`);
```

Comment:
Gilt das wirklich für alle Fehler oder nur für bestimmte?

Reply:
Das gilt für alle Fehler, die vom Update Endpunkt geworfen werden. Wir haben ein direkt mapping vom i18n zu einer klaren Beschreibung. Das ist der einzige Endpunkt im PersonenkontextStore, der in der View hier benutzt wird.